### PR TITLE
Create Mobile UI Section

### PR DESCRIPTION
**Pull request by Topener**
Here's the draft to add a Mobile section to the developer portal

### CODE REVIEW

**Comment by vgarciabnz** [comment]
@Topener I added a commit with more changes. They are not supposed to be final, but they enable us to work in the development of the docs.

- Add "Mobile UI" entry in the sidebar. It is uses "autogenerated" so the structure can be fully defined in the Mobile UI library.
- Add the entry in the migration script to download the mobile-ui folder.
- Changes `git sparse-checkout set` by `git sparse-checkout add` in the migration script. The deploy in netlify has worked, but we should fully understand what is happening before merging it.

File: sidebarsMobile.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    mobile: [
+        { id: 'overview', label: 'Overview', type: 'doc' },
+        {
+            'Mobile UI': [
```
[the remark this answers is not in the document]
Comment (Topener) [reply sidebarsMobile.js:5]: The main issues are "Demo" which should be "Demo Application" and "Api" should be turned into "API".

**Comment by Topener** [comment]
Will the FIGMA link remain public/visible for the foreseeable future?

File: mobile/readme.md
[the remark this answers is not in the document]
Comment (vgarciabnz) [reply mobile/readme.md:1]: We have added a short overview of the resources in the Mobile section, check it out and let me know if it looks good

File: sidebarsMobile.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (vgarciabnz) [reply sidebarsMobile.js:5]: Updated, I have used `title` instead of `sidebar_label`, I think it serves the purpose as well

**Comment by vgarciabnz** [comment]
> Will the FIGMA link remain public/visible for the foreseeable future?

Yes, the link has been updated and it is meant to be public and constant

**Review by vgarciabnz — APPROVED** [review]

**Pull request merged**